### ISSUE TITLE
Click or enter in project list does not work

### ISSUE THREAD

**Issue by jarod51** [alefragnani/vscode-project-manager#27]
Hi

Everything is in the title. When i list my projects and want to load one, click or enter does nothing

Platform : Mac Os X 10.11.3 , VSCode latest version, extensions up to date

**Comment by mhedgpeth** [comment]
I am having the same issue on Windows 7 machine. My extensions are up to date as well.

**Comment by alefragnani** [comment]
Hi, 

Did you defined the `projectManager.codePath` setting, as described in **readme.md**.

**Comment by alefragnani** [comment]
Hi @mhedgpeth . 

Any news about it? Are you still having any problem?

**Comment by mhedgpeth** [comment]
Apologies for the delay, I had to reimage my laptop after replying to the issue. It now works for me on a windows machine. I cloned the repo and see that the `projectManager.codePath` setting defaults to "code" if not defined, which works for windows installations.

**Comment by alefragnani** [comment]
Good to know :+1: 
What about you @jarod51? Are you still having problems?

**Comment by johnemau** [comment]
I am having the same issue- Windows 10, I have tried the following two line in my settings.

```
    // Indicate the full Code app path
	"projectManager.codePath": "code",
```
```
    // Indicate the full Code app path
	"projectManager.codePath": "C:\\Program Files\\Microsoft VS Code\\Bin\\Code.cmd",
```

Any my projects.json looks like this...
```
[
	{
		"label": "Spider Man (1)",
		"description": "c:\\src\\blah1"
	},
	{
		"label": "Iron Man (2)",
		"description": "c:\\src\\blah2"
	},
	{
		"label": "Flash (3)",
		"description": "c:\\src\\blah3"
	}
]
```

Does it look like I am missing anything?

**Comment by alefragnani** [comment]
Hi @johnemau . 

Do you have a _32bits_ or _64bits_ Windows installation? If you have a _64bits_ installation then your path should be `C:\\Program Files (x86)\\Microsoft VS Code\\Bin\\Code.cmd`.

I hope this kind of problem to disappear with the next **VSCode** release, since a new API will be available and this `setting` will not be needed anymore.

Hope this helps.

**Comment by Linrstudio** [comment]
**X**:\\Program Files (x86)\\Microsoft VS Code\\Bin\\Code.cmd
X is your Code app path. Right click VS Code, view Property item.

My VS Code(x64): D:\\Program Files (x86)\\Microsoft VS Code\\Bin\\Code.cmd

**Comment by airstep** [comment]
And what about linux support? Any updates? Still can't open project... ((
Linux archy 4.5.1-1-ARCH #1 SMP PREEMPT Thu Apr 14 19:19:32 CEST 2016 x86_64 GNU/Linux

**Comment by alefragnani** [comment]
Hi @airstep ,
I pretend to update the extension to use the new API released in VSCode version 1.1. I will release the update in the next week. Stay tunned ;-)
